destinations/AppDestination.js

```diff
@@ -178,6 +178,8 @@ export default class AppDestination extends Destination {
       if (data.hasOwnProperty('website')) this.website = data.website
       if (data.hasOwnProperty('tel')) this.tel = data.tel
       if (data.hasOwnProperty('currency')) this.currency = data.currency
+      if (data.hasOwnProperty('saferpay_paymentMethodes'))
+        this.saferpay_paymentMethods = JSON.parse(data.saferpay_paymentMethodes)
```

Comment:
not sure if its better to leave in the typo or remove it for the frontend - I decided for the latter :D